components/api_request_helper/api_request_helper.h

```diff
@@ -197,20 +197,6 @@ class APIRequestHelper {
       const base::flat_map<std::string, std::string>& headers = {},
       const APIRequestOptions& request_options = {});
 
-  using DownloadCallback = base::OnceCallback<void(
-      base::FilePath,
-      const base::flat_map<std::string, std::string>& /*response_headers*/)>;
-
-  // TODO(petemill): Move Download (and OnDownload) to a separate module,
-  // it does not share much from this file, which is meant to be for JSON APIs.
```

Comment:
@petemill @nullhook , I removed download part as PlaylistThumbnailDownloader is the only caller.